flocx_market/tests/unit/conf/test_conf.py

```diff
@@ -1,9 +1,11 @@
 import flocx_market.conf as conf
 import socket
+import pytest
 
 CONF = conf.CONF
 
 
+@pytest.fixture()
```

Comment:
I think you may have misunderstood the use of the `pytest.fixture` decorator.  These aren't fixtures, and you can safely remove the decorator from all the tests.

Reply:
I forgot to remove these. My bad